internal/exec/exec_test.go

```diff
@@ -77,4 +78,51 @@ var _ = Describe("Execute", func() {
 		// wg.Wait()
```

Comment:
comment can be removed, yes?

Reply:
I didn't change this test, and it should probably be improved, but I don't want to get into this now